### DISCUSSION TITLE
How do I disable default accelerators that I don't use?

### DISCUSSION THREAD

**Discussion by WayneYam** [xournalpp/xournalpp#5458] [Q&A]
I don't use some of the default shortcuts like F12, and when I try to bind it to my custom function it doesn't work. Is there a way around this?

**Comment by rolandlo** [comment]
In principle you could do that (for F12) by removing the corresponding line in the file `/usr/share/xournalpp/ui/mainmenubar.xml` (in case you are on Linux).
